readme_assets/system_diagram/system_architecture.xml

```diff
@@ -0,0 +1,133 @@
+            </Array>
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-22" style="rounded=0;orthogonalLoop=1;jettySize=auto;html=1;entryX=0.5;entryY=1;entryDx=0;entryDy=0;startArrow=none;startFill=0;" edge="1" parent="1" source="JVIGxF9oKAkimninJbSp-7" target="JVIGxF9oKAkimninJbSp-4">
+          <mxGeometry relative="1" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-7" value="API Database" style="strokeWidth=1;html=1;shape=mxgraph.flowchart.database;whiteSpace=wrap;fillColor=#dae8fc;strokeColor=#6c8ebf;" vertex="1" parent="1">
+          <mxGeometry x="350" y="210" width="130" height="70" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-23" style="edgeStyle=none;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;entryX=0.5;entryY=1;entryDx=0;entryDy=0;startArrow=classic;startFill=1;endArrow=none;endFill=0;" edge="1" parent="1" source="JVIGxF9oKAkimninJbSp-9" target="JVIGxF9oKAkimninJbSp-4">
+          <mxGeometry relative="1" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-9" value="ElasticSearch" style="strokeWidth=1;html=1;shape=mxgraph.flowchart.database;whiteSpace=wrap;fillColor=#dae8fc;strokeColor=#6c8ebf;" vertex="1" parent="1">
+          <mxGeometry x="690" y="210" width="130" height="70" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-25" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;entryX=0.5;entryY=1;entryDx=0;entryDy=0;entryPerimeter=0;startArrow=classic;startFill=1;endArrow=none;endFill=0;exitX=1;exitY=0.5;exitDx=0;exitDy=0;" edge="1" parent="1" source="JVIGxF9oKAkimninJbSp-10" target="JVIGxF9oKAkimninJbSp-9">
+          <mxGeometry relative="1" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-10" value="API Server Group&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#dae8fc;strokeColor=#6c8ebf;" vertex="1" parent="1">
+          <mxGeometry x="435" y="430" width="300" height="130" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-12" value="Catalog-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" vertex="1" parent="1">
+          <mxGeometry x="455" y="480" width="80" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-13" value="Catalog-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" vertex="1" parent="1">
+          <mxGeometry x="545" y="480" width="80" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-14" value="Catalog-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" vertex="1" parent="1">
+          <mxGeometry x="635" y="480" width="80" height="40" as="geometry" />
+        </mxCell>
```

Comment:
I think 'django-api' might be a better term to use here as "catalog" could be easily confused with our https://github.com/wordpress/openverse-catalog project.

```suggestion
        <mxCell id="JVIGxF9oKAkimninJbSp-12" value="django-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" vertex="1" parent="1">
          <mxGeometry x="455" y="480" width="80" height="40" as="geometry" />
        </mxCell>
        <mxCell id="JVIGxF9oKAkimninJbSp-13" value="django-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" vertex="1" parent="1">
          <mxGeometry x="545" y="480" width="80" height="40" as="geometry" />
        </mxCell>
        <mxCell id="JVIGxF9oKAkimninJbSp-14" value="django-api" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#ffe6cc;strokeColor=#d79b00;" vertex="1" parent="1">
          <mxGeometry x="635" y="480" width="80" height="40" as="geometry" />
        </mxCell>
```